CRM/Report/Form.php

```diff
@@ -3658,7 +3658,12 @@ public function customDataFrom($joinsForFiltersOnly = FALSE) {
       return;
     }
     $mapper = CRM_Core_BAO_CustomQuery::$extendsMap;
-    $customTables = explode(',', CRM_Core_DAO::singleValueQuery("SELECT GROUP_CONCAT(table_name) FROM civicrm_custom_group"));
+    //CRM-18276 GROUP_CONCAT could be used with singleValueQuery and then exploded,
+    //but by default that truncates to 1024 characters, which causes errors with installs with lots of custom field sets
+    $customTablesDAO = CRM_Core_DAO::executeQuery("SELECT table_name FROM civicrm_custom_group", CRM_Core_DAO::$_nullArray);
+    while ($customTablesDAO->fetch()) {
+      $customTables[] = $customTablesDAO->table_name;
```

Comment:
You need to initialise '$customTables = array();' above otherwise it will through notice error like it did
https://test.civicrm.org/job/CiviCRM-Core-PR/8427/testReport/junit/(root)/api_v3_ReportTemplateTest__testReportTemplateGetRowsAllReports/testReportTemplateGetRowsAllReports_with_data_set__0/